webui/src/pages/user/user-settings-namespaces.tsx

```diff
@@ -0,0 +1,337 @@
+                                chosenNamespace={this.state.chosenNamespace}
+                                namespaces={this.state.namespaces}
+                                onChange={this.handleChangeNamespace}
+                            />
+                            <Box className={this.props.classes.memberContainer}>
+                                <Box className={this.props.classes.memberListHeader}>
+                                    <Typography variant='h5'>Members in {this.state.chosenNamespace.name}</Typography>
+                                    <Button className={this.props.classes.addButton} variant='outlined' onClick={this.handleOpenAddDialog}>
+                                        Add Namespace Member
+                                    </Button>
+                                </Box>
+                                <Paper>
+                                    {this.state.members.map(member =>
+                                        <UserNamespaceMember
+                                            key={'nspcmbr-' + member.user.loginName + member.user.provider}
+                                            member={member}
+                                            user={this.props.user}
+                                            service={this.props.service}
+                                            onRemoveUser={() => this.removeUser(member)} />)}
+                                </Paper>
+                            </Box>
+                        </Box>
+                        {this.renderAddDialog()}
+                    </React.Fragment>
+                    : <div>No Namespaces Available. 
+                        Read <Link color='secondary' href='https://github.com/eclipse/openvsx/wiki/Publishing-Extensions' target='_blank'>here</Link> about creating namespaces.
+                    </div>
+            }
+        </React.Fragment>;
+    }
+
+    protected renderAddDialog() {
+        return <React.Fragment>
+            <Dialog open={this.state.addDialogIsOpen} >
+                <DialogTitle id="form-dialog-title">Add User to Namespace</DialogTitle>
+                <DialogContent>
+                    <DialogContentText>
+                        Enter the Login Name of the User you want to add.
+                    </DialogContentText>
+                    <TextField
+                        autoFocus
+                        margin="dense"
+                        id="name"
+                        autoComplete="off"
+                        label="Open VSX User"
+                        fullWidth
+                        onChange={this.handleUserSearch}
+                    />
+                </DialogContent>
+                <DialogActions>
+                    <Button onClick={() => this.doCloseAddDialog()} color="primary">
+                        Cancel
+                    </Button>
+                </DialogActions>
+            </Dialog>
+            <Popper
```

Comment:
The list should hide the logged-in user.

If I search for myself, then click on my result paper, then change the search term to someone else, nothing happens when I click on the new result paper. The dialog seems to get into an invalid state.

<img width="428" alt="Screen Shot 2020-05-05 at 10 20 30" src="https://user-images.githubusercontent.com/4067210/81047125-15431300-8eba-11ea-8624-0437959e380e.png">